pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/legacy/redigerbar/AvslagUfoeretrygd.kt

```diff
@@ -0,0 +1,1696 @@
+package no.nav.pensjon.brev.maler.legacy.redigerbar
+
+import no.nav.pensjon.brev.api.model.TemplateDescription
+import no.nav.pensjon.brev.api.model.maler.Brevkode
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.AvslagUfoeretrygdDto
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.AvslagUfoeretrygdDtoSelectors.PesysDataSelectors.orienteringOmRettigheterUfoere
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.AvslagUfoeretrygdDtoSelectors.PesysDataSelectors.pe
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.AvslagUfoeretrygdDtoSelectors.pesysData
+import no.nav.pensjon.brev.maler.fraser.common.Constants
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.maler.legacy.*
+import no.nav.pensjon.brev.maler.legacy.fraser.*
+import no.nav.pensjon.brev.maler.vedlegg.vedleggDineRettigheterOgPlikterUfoere
+import no.nav.pensjon.brev.model.format
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.RedigerbarTemplate
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.expression.*
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+
+@TemplateModelHelpers
+object AvslagUfoeretrygd : RedigerbarTemplate<AvslagUfoeretrygdDto> {
+
+    override val kode: Brevkode.Redigerbar = Brevkode.Redigerbar.UT_AVSLAG_UFOERETRYGD
+    override val kategori = TemplateDescription.Brevkategori.FOERSTEGANGSBEHANDLING
+
+    override val template = createTemplate(
+        name = kode.name, letterDataType = AvslagUfoeretrygdDto::class, languages = languages(Bokmal, Nynorsk, English), letterMetadata = LetterMetadata(
+            displayTitle = "Vedtak - avslag på uføretrygd",
+            isSensitiv = false,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VEDTAK,
+            brevtype = LetterMetadata.Brevtype.VEDTAKSBREV,
+        )
+    ) {
+        title {
+            text(
+                Bokmal to "Nav har avslått søknaden din om uføretrygd",
+                Nynorsk to "Nav har avslått søknaden din om uføretrygd ",
+                English to "Nav has denied your application for disability benefit",
+            )
+        }
+        outline {
+            val pe = pesysData.pe
+
+            //IF(PE_Vedtaksdata_Kravhode_KravGjelder <> "sok_uu" AND PE_Vedtaksdata_Kravhode_KravGjelder <> "sok_ys" AND PE_Vedtaksdata_Kravhode_KravArsakType <> "endring_ifu" AND FF_GetArrayElement_String(PE_Vedtaksdata_Kravhode_Kravlinjeliste_Kravlinje_Kravlinjetype) = "ut" AND PE_Vedtaksdata_Kravhode_KravGjelder <> "sok_okn_ug") THEN      INCLUDE ENDIF
+            showIf(
+                (pe.vedtaksdata_kravhode_kravgjelder().notEqualTo("sok_uu") and pe.vedtaksdata_kravhode_kravgjelder()
```

Comment:
Kanskje trekkje dette ut i ein hjelpemetode med eit fornuftig namn? Kognitiv last på ca ein million med alle desse forkortingane.

Reply:
Nevermind, såg gjennom resten av brevet også, veldig mange sånne og enda meir komplekse.

Reply:
Ja, dette er hvorfor jeg har plassert brevet i legacy-pakken. Det er et stort behov for å redusere kompleksitet. Det meste av det definitivt helt unødvendig kompleksitet også. Dette er et godt eksempel på hvorfor det tar så lang tid å migrere brevmaler om man skal fikse teknisk gjeld også.